### ISSUE TITLE
[Component page - accordion] focus state not showing and unable to consistently tab to and open accordion on keyboard in Safari

### ISSUE THREAD

**Issue by mollieru** [18F/web-design-standards#534]
*Steps to recreate*
1. click `Option+tab` in Safari to tab through links on the component pages
2. 'Code' accordion doesn't open
3. 'Documentation' accordion doesn't consistently open/close (though sort of does on backwards tab) 

http://screencast-o-matic.com/watch/coQoF9fdvc

*Expected behavior*
Keyboard tabbing should show focus state for 'Code' and 'Documentation' accordion open/close links. 'Enter' should open/close these links.

*Tested on*
OS 10.8
Safari 6.2

**Comment by carodew** [comment]
I'm seeing a similar thing in Chrome. "Enter" works to open/close the code/docs accordion but the headers don't show a focus styling (even when they have focus).

**Comment by carodew** [comment]
Also I'm seeing focus styles appear inconsistently in IE11, although that may just be an artifact of the emulator.

**Comment by mollieru** [comment]
I think this may have something to do w the 'Accordion' component itself, which is also not showing the focus state on tabbing:

(hard to see, but I'm hitting 'tab' when the page is doing nothing at all, sometimes able to open and close hitting 'enter' but w/o focus state, can't see when and where)
http://screencast-o-matic.com/watch/coQo3Zfdzx

**Comment by maya** [comment]
This is fixed in #613.
